alignak/daemons/brokerdaemon.py

```diff
@@ -460,6 +460,8 @@ def setup_new_conf(self):  # pylint: disable=R0915,R0912
             else:
                 name = 'Unnamed broker'
             self.name = name
+            # Set my own process title
+            self.set_proctitle(self.name)
```

Comment:
Yes but the daemon do not yet has its own name (eg. broker-master), only its type (eg. broker)